openapi/components/schemas/calendar/eventCommonFields.yaml

```diff
@@ -0,0 +1,96 @@
+type: object
+properties:
+  userId:
+    $ref: ../../schemas/id.yaml
+  user:
+    type: object
+    description: An alternative to userId for specifying a user.  Supply either 'userId' or the 'user' object stub.
+    properties:
+      id:
+        $ref: ./../../schemas/id.yaml
+  authorId:
+    $ref: ../../schemas/id.yaml
+  author:
+    type: object
+    description: An alternative to authorId for specifying an author.  Supply either 'authorId' or the 'author' object stub.
+    properties:
+      id:
+        $ref: ./../../schemas/id.yaml
+  allDay:
+    oneOf:
+      - type: boolean
+        example: false
+        nullable: false
+      - type: integer
+        example: 0
+        nullable: false
+    description: Whether the event is an all day event.
+  efficacy:
+    type: string
+    enum:
+      - self
+      - series
+    example: self
+    description:
+      Whether the requested change should apply only to the event with the given 'id', or to all events in the
+      series of which event 'id' is a member.
+  title:
+    type: string
+    example: The Yalta Conference
+    description: Title of event.
+  startDate:
+    type: string
+    example: 2024-03-05T19:00:00+10:00
+    description: Start date and time of event in RFC3339 format.
+  endDate:
+    type: string
+    example: 2024-03-05T19:30:00+10:00
+    description: End date and time of event in RFC3339 format.
+  repeat:
+    type: string
+    example: weekly
+    enum: ["", daily, weekly, fortnightly, monthly, yearly]
+    description: Blank for single event.
+  repeat_until:
+    type: string
+    example: 09\/03\/2025 8:00pm
+    description: Date event repeats until
+  type:
```

Comment:
Since this is documenting a JSON endpoint, IMO there's no need to document the integer variant of this field.

Reply:
Okay, I've removed the description of the integer variant of the 'type' field.